Rubberduck.Inspections/Concrete/EmptyIfBlockInspection.cs

```diff
@@ -75,9 +76,24 @@ private bool ContainsExecutableStatements(VBAParser.BlockContext block)
             {
                 foreach (var child in block.children)
                 {
+                    if (child is VBAParser.BlockStmtContext)
+                    {
+                        Debug.Assert(child.ChildCount == 1);
+                        if (child.GetChild(0) is VBAParser.VariableStmtContext ||
+                            child.GetChild(0) is VBAParser.ConstStmtContext)
+                        {
+                            continue;
```

Comment:
I'd know because `Dim` and `Const` aren't executable statements, but this explicit exclusion probably deserves an explanatory comment that says so.